src/utils/__tests__/index.spec.ts

```diff
@@ -149,3 +155,163 @@ describe('getTimeIntervalName', () => {
         expect(getTimeIntervalName(300, intervals)).to.equal('5 minutes');
     });
 });
+
+describe('calculateGranularity', () => {
+    it('should return correct granularity for 1 tick', () => {
+        expect(calculateGranularity(1, 'second')).to.equal(0);
+    });
+    it('should return correct granularity for 1 minute', () => {
+        expect(calculateGranularity(1, 'minute')).to.equal(60);
+    });
+    it('should return correct granularity for 2 minutes', () => {
+        expect(calculateGranularity(2, 'minute')).to.equal(120);
+    });
+    it('should return correct granularity for 3 minutes', () => {
```

Comment:
Do we really need so many almost identical test cases? The chance that it would succeed for say 1,2,3 minutes and fail for 5 minutes is very low but each test takes time to execute what makes our deployment take longer time.

Reply:
@akmal-deriv please clarify what you mean. :) It's just smartcharts repo, not deriv-app, and it's launched very quickly with npm start. And when you run tests there are only 68 test cases in the app in total that are passing within 34ms. Also, having these test cases for each case can help us see which input exactly will result in which output when we use these functions.

Reply:
I don't think adding similar test cases would anyhow improve developers experience, we're adding test cases to make sure the function/component work as expected and adding similar cases doesn't make any difference as they are almost identical. They would either all pass or all fail at the same time. 😄 But it's ok anyways.

Reply:
thank you @akmal-deriv )
yeah, i just want to use these test cases for presentational purpose in order to reduce time for debugging in future ;)